README.md

```diff
@@ -72,7 +72,7 @@ before(() => {
     cy.setupPact('ui-consumer', 'api-provider')
 })
 ```
-### cy.usePactWait([alias] | alias)
+### cy.usePactWait([alias] | alias, providerState)
```

Comment:
All the examples show providerState as non-optional.

it is optional, and will default to `''` as per this change.